src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -41,17 +41,20 @@ public void RelativeSearchPath_Is_Null()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Process.Start not allowed inside AppContainer")]
```

Comment:
I believe that `Process.Start` works in AppContainer just fine. Is the actual problem the limitations of the CoreFX test infrastructure?

Reply:
I did not try running the test on UWP as I believed that from inside an AppContainer you can't start another process.

Reply:
I'll try it out.